include/zephyr/posix/pthread.h

```diff
@@ -293,6 +293,13 @@ static inline int pthread_mutexattr_destroy(pthread_mutexattr_t *m)
 
 #define PTHREAD_BARRIER_SERIAL_THREAD 1
 
+/*
+ *  Barrier attributes - type
+ */
+#define PTHREAD_PROCESS_PRIVATE		0
+#define PTHREAD_PROCESS_PUBLIC		1
+#define PTHREAD_PROCESS_DEFAULT		PTHREAD_PROCESS_PRIVATE
```

Comment:
Ah, sorry - you will need to remove this line because `PTHREAD_PROCESS_DEFAULT` is not a standard identifier.